test/asynchronous/test_client.py

```diff
@@ -1737,10 +1738,7 @@ async def _run(self):
                     await asyncio.sleep(0.001)
 
             def run(self):
-                if _IS_SYNC:
-                    self._run()
-                else:
-                    asyncio.run(self._run())
+                self._run()
```

Comment:
How was this test failing in async?

Reply:
It was trying to call `asyncio.run` while in an active loop since it's an async test, which isn't supported.